mkdocs.yml

```diff
@@ -328,17 +328,12 @@ nav:
       - model/Extension/Properties/cdxPropValue.md
       - model/Extension/Properties/cdxProperty.md
 - annexes:
-  - 'Diffs from Previous Editions': annexes/diffs-from-previous-editions.md
-  - 'Getting started with SPDX 3': annexes/getting-started.md
-  - 'RDF Object Model and Identifier Syntax': annexes/RDF-object-model-and-identifier-syntax.md
-  - 'SPDX License Expressions': annexes/SPDX-license-expressions.md
-  - 'SPDX license list matching guidelines': annexes/license-matching-guidelines-and-templates.md
-  - 'Using SPDX short identifiers in Source Files': annexes/using-SPDX-short-identifiers-in-source-files.md
-  - 'Using SPDX to comply with norms, standards and regulation': annexes/using-SPDX-to-comply-with-industry-guidance.md
-  - 'Including Security Information in SPDX': annexes/including-security-information-in-SPDX.md
-  - 'SPDX Lite': annexes/SPDX-Lite.md
-  - 'Cross-referencing in SPDX 3': annexes/cross-reference.md
-  - 'Package URL specification': annexes/pkg-url-specification.md
+  - 'A: Diffs from Previous Editions': annexes/diffs-from-previous-editions.md
+  - 'B: RDF Object Model and Identifier Syntax': annexes/RDF-object-model-and-identifier-syntax.md
+  - 'C: SPDX License Expressions': annexes/SPDX-license-expressions.md
+  - 'D: SPDX license list matching guidelines': annexes/license-matching-guidelines-and-templates.md
+  - 'E: Package URL specification': annexes/pkg-url-specification.md
```

Comment:
Some annex headings capitalize the first word and lowercase the rest, while the other capitalize (almost) every words. Should stick to one style.

(Exceptions can be made for proper nouns/names)

Reply:
I'm taking the titles from the Annex files themselves:

```
  - 'A: Differences from previous editions': annexes/diffs-from-previous-editions.md
  - 'B: RDF object model and identifier syntax': annexes/RDF-object-model-and-identifier-syntax.md
  - 'C: SPDX license expressions': annexes/SPDX-license-expressions.md
  - 'D: License matching guidelines and templates': annexes/license-matching-guidelines-and-templates.md
  - 'E: Package URL specification v1': annexes/pkg-url-specification.md
```